addons/ieds/CfgVehicles.hpp

```diff
@@ -0,0 +1,130 @@
+                class values {
+                    class Above {
+                        name = CSTRING(Height_AboveGround);
+                        value = 0;
+                        default = 1;
+                    };
+                    class slightly {
+                        name = CSTRING(Height_SlightlyBurried);
+                        value = 1;
+                    };
+                    class medium {
+                        name = CSTRING(Height_MediumBurried);
+                        value = 2;
+                    };
+                    class almost {
+                        name = CSTRING(Height_AlmostBurried);
+                        value = 3;
+                    };
+                    class fully {
+                        name = CSTRING(Height_FullyBurried);
+                        value = 4;
+                    };
+                };
+            };
+
+            class iedActivationType {
+                displayName = "Activation Type";
+                description = "How is the IED activated";
+                typeName = "NUMBER";
+                class values {
+                    class None {name="None"; value=-1; };
+                    class PressurePlate {name="Pressure Plate"; value=0; default=1;};
+                    class Radio {name="Radio"; value=1; };
+                };
+            };
+
+            class activatedForTargets {
+                displayName = "Activated for";
+                description = "What types is the IED activated for";
+                typeName = "NUMBER";
+                class values {
+                    class None {name="None"; value=-1; };
+                    class All {name="Any type"; value=0; default=1;};
+                    class Vehicles {name="Any Vehicle"; value=1; };
+                    class Land {name="Ground Vehicles"; value=2; };
+                    class Air {name="Airial Vehicles"; value=3; };
+                    class Man {name="Man"; value=4; };
+                };
+            };
+
+            class activatedForSides {
+                displayName = "What sides activate this IED";
+                description = "What types is the IED activated for";
+                typeName = "NUMBER";
+                class values {
+                    class None {name="None"; value=-1; };
+                    class All {name="Any side"; value=0; default=1; };
+                    class West {name="BLUFOR"; value=1; };
+                    class East {name="OpFOR"; value=2; };
+                    class Ind {name="Independant"; value=3; };
+                    class Civ {name="Civilian"; value=4; };
+                };
+            };
+        };
+
+        class ModuleDescription {
+            description = CSTRING(ModuleDesc);
+        };
+    };
+
+    class ace_zeus_moduleBase;
+    class GVAR(moduleCreateIedZeus): ace_zeus_moduleBase {
+        curatorCanAttach = 1;
+        displayName = CSTRING(Module);
+        function = QFUNC(moduleZeus);
```

Comment:
^ See above. I don't know on what machine this will be executed. Every machine? The Server? The Zeus-Client?